plaso/dependencies.py

```diff
@@ -145,12 +145,6 @@ def _CheckPythonModule(
       map(int, _VERSION_SPLIT_REGEX.split(minimum_version)))
 
   if module_version_map < minimum_version_map:
-    if not is_required:
```

Comment:
You'll need to re-add these checks since they've not been integrated in l2tdevtools yet